.dockerignore

```diff
@@ -0,0 +1,6 @@
+Dockerfile
+.vscode/
```

Comment:
Could we add `.svelte-kit` and `node_modules` to speed up local development as well ?

Reply:
yes